style.css

```diff
@@ -1,624 +1,995 @@
+  background-color: #fff;
+  color: black;
+  padding: 10px 24px;
+  margin: 0 auto;
+  overflow: hidden;
+}
+
+.recent-works-title {
+  width: 100%;
+  border-bottom: 1px solid #888;
+  height: 100px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  flex-grow: 0;
+  margin: 0;
+}
+
+.recent-works-title h2 {
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 52px;
+  color: #1c1a19;
+}
+
+.recent-work,
+.more-of-my-works {
+  display: flex;
+  flex-direction: column;
+  height: 475px;
+}
+
+.recent-work {
+  margin-top: 50px;
+}
+
+.recent-work-view {
+  background-color: #f55800;
+  width: 50%;
+  height: 40px;
+  text-align: center;
+  text-decoration-line: none;
+  padding: 10px;
+  color: #fff;
+}
+
+.recent-work-view:active {
+  background: #e05304;
+}
+
+.recent-work-view:hover {
+  background: #fff5e1;
+  border: 1px solid #f55800;
+  color: #f55800;
+}
+
+.recent-work-view:disabled {
+  background: #f4f1f0;
+  border: none;
+  color: #5b5957;
+}
+
+.recent-work-view:enabled {
+  background: #f55800;
+  border: 1px solid #f55800;
+  color: #fff;
+}
+
+.more-of-my-works {
+  padding-left: 16px;
+}
+
+.feature-work {
+  display: none;
+}
+
+.tools-used-here {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  padding-left: 16px;
+  gap: 10px;
+  margin-top: 0;
+  margin-bottom: 10px;
+  padding-inline-start: 0;
+  list-style: none;
+}
+
+.first-project-tools {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  padding-left: 16px;
+  gap: 10px;
+  margin-top: 0;
+  margin-bottom: 10px;
+  padding-inline-start: 0;
+  list-style: none;
+}
+
+.view-work {
+  background-color: #f55800;
+  margin-left: -16px;
+  margin-right: -16px;
+  width: calc(100% + 16px);
+  height: 40px;
+  text-align: center;
+  text-decoration-line: none;
+  padding: 10px;
+  color: #fff;
+}
+
+.recent-work-pic {
+  width: 100%;
+  object-fit: contain;
+}
+
+.recent-work-pic img {
+  width: 100%;
+  height: auto;
+  object-fit: fill;
+  display: block;
+}
+
+.recent-work-details {
+  width: 100%;
+  margin: 0;
+}
+
+#work-third {
+  background-image: linear-gradient(179.35deg, rgba(38, 38, 38, 0) 0.85%, rgba(38, 38, 38, 0.9) 84%), url("./images/works/sec-image.png");
+}
+
+#fourth-work {
+  background-image: linear-gradient(179.35deg, rgba(38, 38, 38, 0) 0.85%, rgba(38, 38, 38, 0.9) 84%), url("./images/works/third-image.png");
+}
+
+#fifth-work {
+  background-image: linear-gradient(179.35deg, rgba(38, 38, 38, 0) 0.85%, rgba(38, 38, 38, 0.9) 84%), url("./images/works/first-image.png");
+}
+
+#sixth-work {
+  background-image: linear-gradient(179.35deg, rgba(38, 38, 38, 0) 0.85%, rgba(38, 38, 38, 0.9) 84%), url("./images/works/sec-image.png");
+}
+
+#seventh-work {
+  background-image: linear-gradient(179.35deg, rgba(38, 38, 38, 0) 0.85%, rgba(38, 38, 38, 0.9) 84%), url("./images/works/third-image.png");
+}
```

Comment:
- [ ] [OPTIONAL] On the desktop version, It is better to implement the hover effect on these project card so that when the user hovers on each of them, he/she see the `see project` which is more functional 😃 👍

Reply:
Thank you for your suggestion. I will put this suggestion on hold for now and implement it later since it's marked as optional. @BertrandConxy